dawn/src/dawn/Compiler/DawnCompiler.cpp

```diff
@@ -91,6 +91,10 @@ struct ComputeEditDistance<std::string> {
   }
 };
 
+bool shouldRunPass(const Options& options, bool runSpecificPass) {
```

Comment:
if we use it like this there is no real reason to do a `checkAndPushBack` right? Would we not want to push back directly then?

Reply:
True, and checkAndPushBack and the whole PassManager is being removed in another PR :)